pom.xml

```diff
@@ -70,6 +70,11 @@ THE SOFTWARE.
     </developer>
   </developers>
 
+  <properties>
+    <jenkins.version>1.609</jenkins.version>
+    <java.level>6</java.level>
+  </properties>
+
   <build>
     <plugins>
       <plugin>
```

Comment:
* The version for the `maven-release-plugin` can be obtained from the parent pom
* Change `http` to `https` in the repositories.